scripts/e2e/run-e2e.sh

```diff
@@ -16,3 +16,12 @@ go test -v -timeout 60m -tags e2e ./... >testoutput.txt || {
     cat testoutput.txt
     exit 1
 }
+
+# install with custom tag
+InstallAGIC "custom-ingress-class"
+
+go test -v -timeout 60m -tags e2eingressclass ./... >testoutput.txt || {
```

Comment:
-tags "e2e e2eingressclass"?

Reply:
no, e2eingressclass is enough

Reply:
didn't see your change about separating those two